DIPs/001/README.md

```diff
@@ -0,0 +1,20 @@
+# Minimum Viable Governance
+This document describes the Minimum Viable Governance of Social Dist0rtion Protocol.
+
+The funds of the DAO are managed by the multi signature wallet deployed in the Main Ethereum Network at the address `0x527621278422FFf45A66f7086bAeAC770CF12b69`
+
+The multi signature wallet defines a *set of owners* and a *number of required confirmations* to execute a transaction.
```

Comment:
there are 2 possible threads imposed by the design of the gnosis multisig:
- if `number of required confirmations` == `number of owners`, then if one owner looses his key, the wallet is locked.
- if `number of required confirmations` much lower than `number of owners`, then a minority can collude and steal all funds.

ideas setup would be `number of required confirmations` = `number of owners + 1`. plus a rule to immediately report on lost keys.

Reply:
Good point. I can make a proposal to increase the threshold from 2 to 3 on the multisig wallet. I've also added a paragraph about keys management.